src/navigation/useDeepLinking.js

```diff
@@ -84,13 +96,35 @@ export function useDeepLinkHandler() {
           });
           break;
 
+        case "discover": {
+          const dapp =
+            pathname &&
+            filteredManifests.find(
+              m => pathname?.toLowerCase?.() === `/${m.id}`,
```

Comment:
i think you can remove the `?` here since you already checked for existence before.
Plus not a big fan of the comparison to `/${m.id}`
maybe format the pathname outside of the switch for use in everycase preformated a bit like whats done on the "buy" case `pathname.replace(/(^\/+|\/+$)/g, "");`

Reply:
Done 👌